vagrant/provisioning/arkcase-dev-facts.yml

```diff
@@ -509,5 +509,6 @@ confluent_major_version: 5.4
 confluent_minor_version: 7
 confluent_scala_version: 2.12
 
-enable_elasticsearch: true
-enable_kafka: true
\ No newline at end of file
+# Enable these for Arkcase.NEXT development
+enable_elasticsearch: false
```

Comment:
should we have an arkcase-next-dev-facts.yml?

Reply:
or "arkcase-dev-microservices-facts.yml"?

Reply:
Created  arkcase-dev-microservices-facts.ym. Thanks.